app/javascript/components/patient/Patient.js

```diff
@@ -263,8 +276,10 @@ class Patient extends React.Component {
                 )}
                 {this.props.details.blocked_sms && this.props.details.preferred_contact_method?.includes('SMS') && (
                   <span className="font-weight-bold text-danger">
-                    {this.props.details.preferred_contact_method}
-                    <InfoTooltip tooltipTextKey="blockedSMSContactMethod" location="top" />
+                    {this.props.details.preferred_contact_method || '--'}
+                    <Form.Label className="tooltip-whitespace">
```

Comment:
the blocked SMS stuff was refactored in my PR, seems like a rebase reverted those changes back